### PR TITLE
embed real, full context in processor previews

### PR DESCRIPTION
**Pull request by unity**

### CODE REVIEW

File: server/middlewares/fetch-user.js [hunk 1]
```diff
@@ -62,15 +61,7 @@ function getEventsForUserId(client, user_id) {
...
-              context: {
-                location: {
-                  latitude: location.lat,
-                  longitude: location.lon
-                },
-                page: {
-                  url: context.page_url
-                }
-              }
+              context
```
Comment (timliuhull) [inline server/middlewares/fetch-user.js]: What's in context? Are we sure it's ok to return the whole thing?

File: server/middlewares/fetch-user.js [hunk 2]
```diff
@@ -42,7 +42,6 @@ function getEventsForUserId(client, user_id) {
               source,
               type
             } = e;
-            const { location = {} } = context;
```
Comment (timliuhull) [inline server/middlewares/fetch-user.js]: this is the syntax for making location an empty object if it doesn't exist in the context I guess?

File: server/middlewares/fetch-user.js (hunk 1 above)
Comment (unity) [reply server/middlewares/fetch-user.js]: It's the return object from the User Events we index. It's an object that has a fixed shape: https://github.com/hull/hull/commit/664019fd275b532c370b99ff475197ae09f3c2b0

File: server/middlewares/fetch-user.js (hunk 2 above)
Comment (unity) [reply server/middlewares/fetch-user.js]: exactly :) https://wesbos.com/destructuring-default-values/

**Pull request merged**